server/models/Fiddle.ts

```diff
@@ -0,0 +1,33 @@
+import mongoose = require("mongoose");
+
+const FiddleSchema = new mongoose.Schema(
+    {
+      _id:  String,
+      creator: String,
+      description: String,
+      fiddle: String,
+      name: String,
+      private: Number,
+      stars: Number,
```

Comment:
Perhaps we should make this an array of objects `{ _id: 1234, starredOn: 1515134542 }` or something like that, what do you think? It will allow us to track who starred what.

Reply:
Yeah sounds good. 

We could raise an issue for this with something like 'update swagger.json to show 'stars' field is an array of objects' and we could even label the issue 'first timers only' and wait for a week or so and see if anyone picks it up. 

I think we need to start raising more 'first timers only' issues to encourage people to participate.

I certainly looked at the 'first timers only' site when I initially looked into getting started with open source contribution. However a lot of the issues were for older projects that people hadn't been contributing to for months even years.

Reply:
Yeah good idea, I think for this PR though we should address this.
Great idea though, I will make sure to raise all future issues as more bite-sized and beginner friendly